modules/camera/camera_macos.h

```diff
@@ -28,9 +28,6 @@
 /* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                 */
 /**************************************************************************/
 
-#ifndef CAMERA_MACOS_H
-#define CAMERA_MACOS_H
```

Comment:
This header is included in both Obj-C code and C++ code - `register_types.cpp`, so won't work like this.